lib/rollbar/configuration.rb

```diff
@@ -44,6 +44,7 @@ class Configuration
     attr_accessor :uncaught_exception_level
     attr_accessor :scrub_headers
     attr_accessor :sidekiq_threshold
+    attr_accessor :thread_timeout
```

Comment:
This name is confusing. Also, I'm not sure this should be something configurable. Maybe we can just use a constant for this.